Rubberduck.Inspections/QuickFixProvider.cs

```diff
@@ -170,7 +170,19 @@ public void FixAll(IQuickFix fix, Type inspectionType, IEnumerable<IInspectionRe
                 var modules = filteredResults.Select(s => s.QualifiedSelection.QualifiedName).Distinct();
                 foreach (var module in modules)
                 {
-                    _state.GetRewriter(module).Rewrite();
+                    var moduleRewriter = _state.GetRewriter(module);
+                    if (moduleRewriter.IsDirty)
```

Comment:
this seems like a weird name for a flag indicating that the module rewriter wants to perform actions. I'd expect `IsDirty` to force a refresh instead... maybe `HasActions` or something along these lines could be less of a tripwire